sketch.js

```diff
@@ -48,27 +49,40 @@ var albumWordCount = function(anAlbum, aWord) {
   return wordCount;
 }
 
-// Example accessing songs in first album:
-// console.log(discography[0].name);
-// console.log('  mentions of "you": ', albumWordCount(discography[0].songs, "we"));
-// console.log('  mentions of "me": ', albumWordCount(discography[0].songs, "me"));
-
-// Another sample to see this in every album
-var countsInAllAlbums = function(aDiscography) {
-  for (var i = 0; i < aDiscography.length; i++) {
-    console.log(aDiscography[i].name)
-    console.log('  mentions of "we": ', albumWordCount(aDiscography[i].songs, "we"));
-    console.log('  mentions of "me": ', albumWordCount(aDiscography[i].songs, "me"));
+// returns an object of objects with counts
```

Comment:
I'll explain it after class. But you can use the output, see line [75](https://github.com/sukanye/Infographic-Round2/pull/2/files/80ce28c94a6666273a393f01e5d5aa84d26d7150#diff-9f2094443505273ff51ea1d1702e4367R75)